FE-service/src/components/subpages/arena/ArenaPage.jsx

```diff
@@ -0,0 +1,136 @@
+import { useState, useEffect } from "react";
+import { CircularProgress, Typography } from "@mui/material";
+import {
+  ArenaContainer,
+  ArenaContent,
+  PokemonListContainer,
+  PokemonListWrapper,
+  PokemonCardContainer,
+  FightButtonContainer,
+  FightButton,
+  EmptyCardImage,
+  TrashIcon,
+  EmptyCardContainer,
+  SummaryContainer,
+} from "./ArenaPage.styles";
+import TrashImage from "../../../icons/trash.png";
+import PokeballImage from "../../../icons/pokeball.png";
+import PokemonCard from "../../shared/pokemonCard/PokemonCard";
+import { useTheme } from "../../../context/ThemeContext";
+import WinnerModal from "./WinnerModal";
+import useHandleFight from "./useHandleFight";
+import { enqueueSnackbar } from "notistack";
+import useGetDbData from "../../../hooks/useGetDbData";
+
+const ArenaPage = () => {
+  const {
+    data: arenaPokemonData,
+    loading: arenaLoading,
+    error: arenaError,
+  } = useGetDbData("arenaPokemons");
+  const {
+    data: arenaFightsData,
+    loading: fightsLoading,
+    error: fightsError,
+  } = useGetDbData("pokemons");
+
+  const [selectedPokemons, setSelectedPokemons] = useState([]);
+  const { theme } = useTheme();
+
+  useEffect(() => {
```

Comment:
It is good practice to place the `useEffect` hook at the very bottom of the component.